### PR TITLE
Fix UserGuide.md

### PR DESCRIPTION
**Pull request by yyyaohhh**
#206

### CODE REVIEW

File: docs/images/Ui.png
Comment (xCOLOURx) [inline docs/images/Ui.png:1]: I think this is not the actual sample data, the actual one that is loaded is in SampleDataUtil.java which should be in 3 different semesters.

**Review by xCOLOURx — CHANGES_REQUESTED** [review]
Maybe we should have a different image for the start up picture like Startup.png and keep the previous Ui.png to show more information.

The rest LGTM 👍

File: docs/images/Ui.png
Comment (rocketninja7) [reply docs/images/Ui.png:1]: Might need to delete moduleplan.json in data folder

**Comment by yyyaohhh** [comment]
> Maybe we should have a different image for the start up picture like Startup.png and keep the previous Ui.png to show more information.
The previous Ui.png does show more information, but there are more other images throughout the UserGuide that explain the features while showing more information, so is it necessary to put the previous Ui.png at the start of the UG?

File: docs/images/Ui.png
Comment (yyyaohhh) [reply docs/images/Ui.png:1]: updated accordingly 👍

**Comment by xCOLOURx** [comment]
> > Maybe we should have a different image for the start up picture like Startup.png and keep the previous Ui.png to show more information.
> 
> The previous Ui.png does show more information, but there are more other images throughout the UserGuide that explain the features while showing more information, so is it necessary to put the previous Ui.png at the start of the UG?

Oh I was thinking because others like index.md also use UI.png so for user guide instead of showing UI.png we display another one which is the startup.png u added. So for Userguide thr will be no more UI.png and its startup.png instead.

**Comment by yyyaohhh** [comment]
> > > Maybe we should have a different image for the start up picture like Startup.png and keep the previous Ui.png to show more information.
> > 
> > 
> > The previous Ui.png does show more information, but there are more other images throughout the UserGuide that explain the features while showing more information, so is it necessary to put the previous Ui.png at the start of the UG?
> 
> Oh I was thinking because others like index.md also use UI.png so for user guide instead of showing UI.png we display another one which is the startup.png u added. So for Userguide thr will be no more UI.png and its startup.png instead.

Ok understand now 👍 
Updated accordingly

**Review by xCOLOURx — APPROVED** [review]
LGTM 👍

**Pull request merged**